kind/kind.sh

```diff
@@ -48,42 +42,24 @@ function _wait_kind_up {
     kubectl wait -n kube-system --timeout=12m --for=condition=Ready -l k8s-app=kube-dns pods
 }
 
-function _fetch_kind() {
-    mkdir -p "${KIND_DIR}"
-    KIND="${KIND_DIR}"/.kind
-    if [ -f "$KIND" ]; then
-        current_kind_version=$($KIND --version | awk '{print $3}')
-    fi
-    if [[ $current_kind_version != $KIND_VERSION ]]; then
```

Comment:
can we move this validation to `verify.sh tools` instead? This allows everyone to validate if the versions of the binaries in path are good!